packages/playground/website/public/.htaccess

```diff
@@ -1,2 +1,5 @@
 RewriteEngine on
 RewriteRule ^plugin-proxy$ plugin-proxy.php [NC]
+
+RewriteCond %{QUERY_STRING} start_button=1 [NC]
+RewriteRule ^$ button.php [L]
```

Comment:
comment?

does this trap `start_button=1` requests or _add_ `start_button=1` to requests?
does this redirect happen by default?